<?xml version="1.0" ?>
<launch>
    <arg name="config_file"/>
    <arg name="config_name"/>

    <node name="robot_reach_study_node" pkg="reach_core" type="robot_reach_study_node" output="screen" required="true">
      <rosparam command="load" file="$(arg reach_object)"/>
      <param name="config_name" value="$(arg config_name)"/>
    </node>
</launch>
